libs/src/services/solanaWeb3Manager/transactionHandler.js

```diff
@@ -52,10 +52,10 @@ class TransactionHandler {
    * @returns {Promise<HandleTransactionReturn>}
    * @memberof TransactionHandler
    */
-  async handleTransaction (instructions, errorMapping = null, recentBlockhash = null, logger = console, skipPreflight = null) {
+  async handleTransaction ({ instructions, errorMapping = null, recentBlockhash = null, logger = console, skipPreflight = null }) {
```

Comment:
should skipPreflight just be `false` here?

Reply:
TransactionHandler also has a skipPreflight as part of it's constructor, so we need whatever is passed here to overwrite that one on a per request basis. But should also include that one in _relayTransaction like we do in _locallyConfirmTransaction

Reply:
I mean should the default just be false rather than null?